src/pages/[platform]/build-a-backend/auth/manage-user-session/index.mdx

```diff
@@ -138,6 +130,49 @@ async function currentSession() {
 </Block>
 </BlockSwitcher>
 
+### Refresh session
+
+The `fetchAuthSession` API will auto refresh the user's session when tokens have expired and there is a valid `refreshToken`. Additionally, you can also refresh the session by calling the `fetchAuthSession` API with the `forceRefresh` flag enabled. 
+
+```ts
+import { fetchAuthSession } from 'aws-amplify/auth';
+
+async function currentSession() {
+  try {
+    const { tokens } = await fetchAuthSession({ forceRefresh: true });
+    console.log(tokens);
+  } catch (err) {
+    console.log(err);
+  }
+}
+```
+
+### Refresh session with social providers
+
+To refresh the user's session when authenticating with a social provider, you need to use the `code grant` oauth flow in your Amplify configuration.
+
+```TypeScript
+Amplify.configure({
+   Auth: {
+    Cognito: {
+      ...cognitoConfig,
+      loginWith: {
+        oauth: {
+          ...oauthConfig,
+          responseType: "code",
+        },
+      },
+    },
+  },
+})
+```
+
+<Callout>
+
+**Note:** refresh session with social providers will not work when the user has authenticated with a `token` oauth flow, e.g `{ responseType: 'token' }`.
```

Comment:
```suggestion
**Note:** `forceRefresh` with social providers will not work when the user has authenticated with a `token` OAuth flow, e.g `{ responseType: 'token' }`.
```

Reply:
It is true that `forceRefresh` will not work under those circumstances, but `fetchAuthSession` will also not be able to automatically refresh the session even though the `forceRefresh` option is not passed.

Reply:
Ah got it! Okay, discard this one, but maybe still change oauth to OAuth